tensorboardX/summary.py

```diff
@@ -87,6 +87,9 @@ def hparams(hparam_dict=None, metric_dict=None):
 
     ssi = SessionStartInfo()
     for k, v in hparam_dict.items():
+        if v is None:
+            continue
+
```

Comment:
Thanks for changing this, does that really solve the problem though? 
`None` should also be logged right?